src/core/server/elasticsearch/elasticsearch_service.mock.ts

```diff
@@ -103,6 +124,7 @@ export const elasticsearchServiceMock = {
   create: createMock,
   createInternalSetup: createInternalSetupContractMock,
   createSetup: createSetupContractMock,
+  createStart: createStartContractMock,
```

Comment:
Do we want to also add `createInternalStart` for consistency?

Reply:
We don't have an `InternalElasticsearchServiceStart` type, so I don't think it's useful.